src/query/functions-v2/src/scalars/control.rs

```diff
@@ -140,4 +150,58 @@ pub fn register(registry: &mut FunctionRegistry) {
             }),
         }))
     });
+    registry.register_1_arg_core::<NullableType<GenericType<0>>, BooleanType, _, _>(
+        IS_NULL,
+        FunctionProperty::default(),
+        |_| None,
+        |arg, _| match &arg {
+            ValueRef::Column(NullableColumn { validity, .. }) => {
+                let bitmap = Bitmap::from_iter(validity.iter().map(|v| !v));
+                Ok(Value::Column(bitmap))
+            }
+            ValueRef::Scalar(None) => Ok(Value::Scalar(true)),
+            ValueRef::Scalar(Some(_)) => Ok(Value::Scalar(false)),
+        },
+    );
+    registry.register_1_arg_core::<NullType, BooleanType, _, _>(
+        IS_NULL,
+        FunctionProperty::default(),
+        |_| None,
+        |arg, _| match &arg {
+            ValueRef::Column(len) => {
+                let vec = vec![0xff_u8; (*len).saturating_add(7) / 8];
+                let bitmap = MutableBitmap::from_vec(vec, *len);
```

Comment:
`constant_bitmap(*len).into()`

Reply:
Or just `Ok(Value::Scalar(false))` ?

Reply:
Ok, I will modify it. And How to implement this https://github.com/datafuselabs/databend/pull/7282#discussion_r954312859 , I will revise together.